osi_sensorinputconfiguration.proto

```diff
@@ -89,6 +97,8 @@ message SensorInputConfiguration
     //
     // This determines the limit of the cone of interest of ground truth
     // that the simulation environment has to provide.
+    // Viewing range: [-field_of_view_vertical/2, field_of_view_vertical]
+    // in sensor coordinate system.
```

Comment:
It seems that the range should be defined as:
``[-field_of_view_vertical/2, field_of_view_vertical/2]``
Moreover, I would concretize the definition as follows:
``Viewing range: [-field_of_view_vertical/2, field_of_view_vertical/2]
elevation in the sensor frame at zero azimuth as defined in Spherical3d.``